chapters/introduction.tex

```diff
@@ -68,12 +68,37 @@ \section{Some Definitions}\label{some-definitions}
 \end{definition}
 
 \begin{definition}[Flattening]\index{flattening}
-The translation of a model described in Modelica to the corresponding model described as a hybrid DAE, involving expansion of inherited base classes, parameterization of base classes, local classes
+The translation of a model described in Modelica to the corresponding model described as a hybrid DAE (see \cref{modelica-dae-representation}), involving expansion of inherited base classes, parameterization of base classes, local classes
 and components, and generation of connection equations from \lstinline!connect!-equations (basically, mapping the hierarchical structure of a model into a set of differential, algebraic and discrete equations together with the corresponding variable declarations and function definitions from the model).
 \end{definition}
 
+\begin{definition}[Initialization]\index{initialization}
+Simulation starts with solving the initialization problem at the starting time, resulting in values for all variables that are consistent with the result of the flattening.
+\end{definition}
+
+\begin{definition}[Transient analysis]\index{transient analysis}\index{initial value problem!see{transient analysis}}
+Starting from the result of the initialization problem, the model is simulated forward in time.
+This uses numerical methods for handling the hybrid DAE, resulting in solution trajectories for the model's variables, i.e., the value of the variables as a function of time.
+\end{definition}
+
+\begin{definition}[Simulation]\index{simulation}
+For a model simulation is the combination of the initialization followed by the transient analysis.
+\end{definition}
+
+\begin{definition}[Translation]\index{translation}
+Translation is the process of preparing a Modelica simulation model for simulation, starting with flattening but not including the simulation itself.
+\end{definition}
+
+\begin{nonnormative}
+The model can be analyzed in ways other than simulation, e.g., linearization, and parameter estimation, but they are not described in the specification.
+
+In the numerical literature transient analysis is often called solving the \firstuse{initial value problem}, but that term is not used here to avoid confusion with the initialization problem.
+
```

Comment:
Maybe we should just move this non-normative part to a new non-normative part right below the definition of _transient analysis_, so that it isn't missed when reading the definition of _transient analysis_?

Reply:
Done.